examples/custom-cart-edit/app/lib/fragments.ts

```diff
@@ -0,0 +1,126 @@
+// NOTE: https://shopify.dev/docs/api/storefront/latest/queries/cart
```

Comment:
Are these fragments not already built into the skeleton?

Reply:
The first one is part of skeleton, I added a second query. Maybe I will move it to a new file instead.

Reply:
I edited some fields so we can display options